wandb/sdk/internal/progress.py

```diff
@@ -59,7 +59,7 @@ def read(self, size=-1):
         return bites
 
     def rewind(self) -> None:
-        self.callback(0, -self.bytes_read)
```

Comment:
These arguments are in the wrong order. Compare to `self.read()` above: the first argument is clearly the number of _new_ bytes, the second is clearly the number of _total_ bytes read.

Reply:
Wow, fun 🙃

Reply:
Yikes